rebasehelper/utils.py

```diff
@@ -101,7 +101,15 @@ class ConsoleHelper(object):
 
     @classmethod
     def should_use_colors(cls, conf):
-        """Determine whether ansi colors should be used for CLI output"""
+        """Determine whether ansi colors should be used for CLI output
+
+        Args:
+            conf: configuration object with arguments from the command line
```

Comment:
There is no type (should be `rebasehelper.config.Config`), and the description should start with capital `C` and end with a `.`.